### PR TITLE
feat: Removed Requests Screen from Superadmin Panel

### LINKED ISSUE #1375: Remove Requests Screen from Superadmin Panel
**Is your feature request related to a problem? Please describe.**
The use of Requests screen on Superadmin panel is redundant because, it only shows the requests from the admins who have isAdminApproved attribute as false. Since the new flow goes like this, that a superadmin will create a normal user an admin, we no longer require this screen, so this screen needs to be removed along with its specific corresponding assets (Like icons).

Dashboard _ REQUESTS

**Describe the solution you'd like**
Remove this screen

**Describe alternatives you've considered**
NA

**Approach to be followed (optional)**
NA

**Additional context**
NA

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by KshitijTodkar48**
**What kind of change does this PR introduce?**
 Feature.

**Issue Number:**

Fixes #1375

**Did you add tests for your changes?**
 No.

**Snapshots/Videos:**
Screenshot (98)

**Summary**
This PR solves the issue #1375 by removing the request screen from Super-admin Panel along with its specific corresponding assets.

**Does this PR introduce a breaking change?**
 No.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
 Yes.

### CODE REVIEW

File: src/components/LeftDrawer/LeftDrawer.tsx [hunk 1]
```diff
@@ -5,7 +5,6 @@ import { useHistory } from 'react-router-dom';
 import { ReactComponent as AngleRightIcon } from 'assets/svgs/angleRight.svg';
 import { ReactComponent as LogoutIcon } from 'assets/svgs/logout.svg';
 import { ReactComponent as OrganizationsIcon } from 'assets/svgs/organizations.svg';
-import { ReactComponent as RequestsIcon } from 'assets/svgs/requests.svg';
```
Comment (rishav-jha-mech) [inline src/components/LeftDrawer/LeftDrawer.tsx:8]: Check whether this icon is used somewhere else or not, if not then remove this svg file too

Comment (KshitijTodkar48) [reply src/components/LeftDrawer/LeftDrawer.tsx:8]: Ok, I'll do that.

**Comment by palisadoes** [comment]
Please fix the conflicting files

**Comment by KshitijTodkar48** [comment]
@rishav-jha-mech @palisadoes I am opening a new PR for this issue.

**Comment by palisadoes** [comment]
Please reopen this PR. There is no reason to close it

1. We have already signed reviewers
1. It's against the correct branch
1. The history of activities to help out reviewers is easily visible

Please be considerate of our volunteers time

Do not open another one. Reopen this one.

**Comment by KshitijTodkar48** [comment]
@palisadoes , Sorry for the inconvenience, while solving the conflicts, I messed up some git-workflow. So just to be on safer side, I deleted that branch completely. I have already opened a new PR for this. I'll take care to not repeat such an action in the future. Please let it go this time. 🥹

**Pull request closed without merging**